app/points/calculate-community-tier-workload-points.js

```diff
@@ -0,0 +1,15 @@
+const calculatePointsForTier = require('../../app/points/calculate-points-for-tier')
+
+module.exports = function (communityTiers, caseTypeWeightings, commTierCPEnabled, commTier3AEnabled) {
+  var communityWorkloadPoints = 0
+  communityTiers.getTierCountsAsList().forEach(function (tier) {
+    communityWorkloadPoints += calculatePointsForTier(tier.tierCount, tier.points, caseTypeWeightings)
+  }, this)
+  if (commTierCPEnabled) {
+    communityWorkloadPoints += calculatePointsForTier(communityTiers.a1.tierCount, communityTiers.a1.points, caseTypeWeightings.commTierCPWeighting)
+  }
+  if (commTier3AEnabled) {
+    communityWorkloadPoints += calculatePointsForTier(communityTiers.a3.tierCount, communityTiers.a3.points, caseTypeWeightings)
```

Comment:
According to the docs if tier 3a is enabled then we should be calculating it in the place of tier a. This means we need to know this before we do the initial calculation above and substitute the tier a value appropriately.